circuit_maintenance_parser/parsers/turkcell.py

```diff
@@ -0,0 +1,69 @@
+"""Turkcell parser."""
+import logging
+import re
+from typing import Dict
+
+from bs4.element import ResultSet  # type: ignore
+from dateutil import parser
+
+from circuit_maintenance_parser.errors import ParsingError
+from circuit_maintenance_parser.parser import Html, Impact, CircuitImpact, Status
+
+# pylint: disable=too-many-nested-blocks, too-many-branches
+
+logger = logging.getLogger(__name__)
+
+
+class HtmlParserTurkcell1(Html):
+    """Notifications Parser for Turkcell notifications."""
+
+    def parse_html(self, soup, data_base):
+        """Execute parsing."""
+        data = data_base.copy()
+        try:
+            self.parse_tables(soup.find_all("table"), data)
+            return [data]
+
+        except Exception as exc:
+            raise ParsingError from exc
+
+    def parse_tables(self, tables: ResultSet, data: Dict):
+        """Parse tables."""
+        # Main table
+        td_elements = tables[0].find_all("td")
+        for idx, td_element in enumerate(td_elements):
+            if "Dear Customer" in td_element.text.strip():
+                if "planned" in td_element.text.strip():
+                    data["status"] = Status["CONFIRMED"]
+                else:
+                    data["status"] = Status["CONFIRMED"]
```

Comment:
Why have an if/else block that does the same thing in both branches? Is one of these Status values incorrect?

Reply:
I understood that he only found the "planned" use case, and he was enforcing the default one, that is the same.
Ideally, we should have some `elif` in between when more use cases are added...
is this accurate Kristian?

Reply:
Yeah that is correct Christian. I need more examples with different status. Will be adding more in the future